jbot/src/main/java/me/ramswaroop/jbot/core/slack/models/User.java

```diff
@@ -9,6 +9,8 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class User {
     private String id;
+    @JsonProperty("team_id")
+    private String teamId;    
```

Comment:
If you're sure you get `team_id` in response of user_change event, i can merge it right away.